modules/navArr.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable */
+
+export const list = [
+  {
+    linkId: 'list',
+    pageId: 'listpage',
+  },
+  {
+    linkId: 'add-new',
+    pageId: 'addpage',
+  },
+  {
+    linkId: 'contact-page',
+    pageId: 'contactpage',
+  },
+];
```

Comment:
- [OPTIONAL] It's not a good practice to disable the `eslint` using inline comments, So I suggest removing the first line  `/* eslint-disable */` and Try to fix the linter issue instead 👍

> To Fix This Issue, you've to use `export default` instead of `export` 💡
> For Example
```js
const list = [
  {
    linkId: 'list',
    pageId: 'listpage',
  },
  {
    linkId: 'add-new',
    pageId: 'addpage',
  },
  {
    linkId: 'contact-page',
    pageId: 'contactpage',
  },
];

export default list;

```

> And To import this List 💡 
```js
import list from './navArr.js';
```